changelogs/fragments/68-refactor-save-load-config.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - module_utils - Refactor save_config_to_disk and load_config_to_runtime.
```

```suggestion
  - module_utils - Refactor save_config_to_disk and load_config_to_runtime (https://github.com/ansible-collections/community.proxysql/pull/78).
```

add link to the pull request.

---

@markuman ++